docs/_Aug_milestone_.md

```diff
@@ -0,0 +1,24 @@
+## August Milestone
+
+### Overview
+
+- Finish hasManyThrough relation MVP
+- Continue with docs improvements
+
+### Goals
+
+- [ ] hasManyThrough relation MVP
+
+  - [ ] docs: add HasManyThrough relation #5835
+  - [ ] feat: set up HasManyThrough relation through the CLI #5836
+
+- [ ] Documentation improvement
+
+  - [ ] Rename "legacy juggler bridge" mentioned in docs #5558
+  - [ ] Add Couchdb connector docs #5267
+  - [ ] [APIDocs] Table not showing properly when it includes code snippet #5756
+
```

Comment:
What's our plan for the following tasks? Should/can we include some of them in August backlog?

- docs: untangle connector reference into reference, how-to guides and tutorials #5786
- docs: find a better place for "Extending LoopBack 4" in the sidebar [SPIKE] #5785
- docs: move recipes to how-to guides #5783
- docs: move API references to tsdoc #5778
- docs: improve the structure of "Usage scenarios" (how-to guides) sidebar #5777
- docs: improve the structure of "Behind the scenes" sidebar + find a better name #5776

Reply:
Personally, I would start with improving the sidebar structure (#5776 and #5777), so to prepare places where we will be adding new pages as part of extracting how-to guides from other content.

Reply:
> Personally, I would start with improving the sidebar structure (#5776 and #5777), so to prepare places where we will be adding new pages as part of extracting how-to guides from other content.

+1. it seems like other issues in the epic depends on it. Could you please add acceptance criteria for #5776 and #5777? Thanks. 

I've created an epic #5960 for #5786.  Propose to add #5961 (spike) to Aug.